.github/json_matrices/supported-languages-versions.json

```diff
@@ -6,8 +6,8 @@
     },
     {
         "language": "python",
-        "versions": ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"],
-        "always-run-versions": ["3.8", "3.13"]
+        "versions": ["3.9", "3.10", "3.11", "3.12", "3.13"],
```

Comment:
Please do changes for python and node in different PRs - it is easier to track and test